minikube/bootstrap.sh

```diff
@@ -140,6 +262,10 @@ case "$OS" in
     # TODO: Hyperkit driver is still not stable enough. Use with later releases.
     #installHyperkitDriver
     ;;
+  linux)
+    checkIfInstall
+    upgradeLinuxSoft
```

Comment:
@alnikyur it's better not to split this actions, but rather do install-or-upgrade action for each software at once.